releases/2.13.7.md

```diff
@@ -0,0 +1,66 @@
+The Scala team at Lightbend is pleased to announce the availability of Scala 2.13.7.
+
+### Align with Scala 3
+
+* Update TASTy reader to support Scala 3.1 ([#9791](https://github.com/scala/scala/pull/9791) by [@bishabosha](https://github.com/bishabosha))
+* Allow `import x.{*, given}` under `-Xsource:3` ([#9724](https://github.com/scala/scala/pull/9724) by [@smarter](https://github.com/smarter))
+* Allow `case` in pattern bindings even without `-Xsource:3` ([#9721](https://github.com/scala/scala/pull/9721) by [@smarter](https://github.com/smarter))
+* Deprecate top-level wildcard type parameters ([#9712](https://github.com/scala/scala/pull/9712) by [@som-snytt](https://github.com/som-snytt))
+
+### JDK and Java compatibility
+
+* Support JDK 18 ([#9702](https://github.com/scala/scala/pull/9702) by [@SethTisue](https://github.com/SethTisue))
+* Support JDK 16 records in Java sources ([#9551](https://github.com/scala/scala/pull/9551) by [@harpocrates](https://github.com/harpocrates))
+* Allow concrete private interface methods in Java sources ([#9748](https://github.com/scala/scala/pull/9748) by [@dengziming](https://github.com/dengziming))
+* Use `StringConcatFactory` for string concatenation on JDK 9+ ([#9556](https://github.com/scala/scala/pull/9556) by [@harpocrates](https://github.com/harpocrates))
+
+### Android compatibility
+
+* Add `ClassValueCompat` to support systems without `java.lang.ClassValue` (such as Android) ([#9752](https://github.com/scala/scala/pull/9752) by [@nwk37011](https://github.com/nwk37011))
+ * For Android compatibility, make `Statics.releaseFence()` also catch `NoSuchMethodException` for `java.lang.invoke.VarHandle.releaseFence()` call ([#9739](https://github.com/scala/scala/pull/9739) by [@nwk37011](https://github.com/nwk37011))
+
+### Concurrency
+
+* Fix asymmetric failure behavior of `Future#{zip,zipWith,traverse,sequence}` by making them fail fast regardless of ordering ([#9655](https://github.com/scala/scala/pull/9655) by [@lihaoyi](https://github.com/lihaoyi))
+
+### Collections
+
+* Make `ArrayBuffer`'s iterator fail fast when buffer is mutated ([#9258](https://github.com/scala/scala/pull/9258) by [@NthPortal](https://github.com/NthPortal))
+* Fix `ArrayOps` bugs (by avoiding `ArraySeq#array`, which does not guarantee element type) ([#9641](https://github.com/scala/scala/pull/9641) by [@som-snytt](https://github.com/som-snytt))
+* Deprecate `IterableOps.toIterable` ([#9774](https://github.com/scala/scala/pull/9774) by [@lrytz](https://github.com/lrytz))
+
+### Other changes
+
+* Accept supplementary Unicode characters in identifiers ([#9687](https://github.com/scala/scala/pull/9687) by [@som-snytt](https://github.com/som-snytt))
+* Add `-Vimplicits-max-modules` to control how types are printed in `-Vimplicits` messages ([#9767](https://github.com/scala/scala/pull/9767) by [@mrdziuban](https://github.com/mrdziuban))
```

Comment:
@SethTisue this PR ended up actually changing the behavior to print full type names unless the type is in the pre-defined list of shorthands, as suggested in https://github.com/scala/scala/pull/9767#issuecomment-931125211, so there's not a new `-Vimplicits-max-modules` option. Apologies, I should have changed the name of the PR after addressing the comment

Reply:
@mrdziuban Thanks, good catch! I've removed this bullet from the release notes. Do please change the Scala PR title and description to match the real change